config/application.rb

```diff
@@ -37,5 +40,17 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.to_prepare do
+      # Allows us to use decorator files, which change methods or behavior on upstream classes
+      # with minimal overrides or fuss. Pattern adapted from Spree and Refinery projects
+      Dir.glob(File.join(File.dirname(__FILE__), "../app/**/*_decorator*.rb")).sort.each do |c|
+        Rails.configuration.cache_classes ? require(c) : load(c)
+      end
+
+      Dir.glob(File.join(File.dirname(__FILE__), "../lib/**/*_decorator*.rb")).sort.each do |c|
+        Rails.configuration.cache_classes ? require(c) : load(c)
+      end
+    end
```

Comment:
interpret_visibility_actor and child_relationships_job. We use this decorator pattern whenever we override a file from a gem dependency. This makes it easier to target the exact change we want instead of copying the whole file over.